ui/mirage/scenarios/default.js

```diff
@@ -2,6 +2,7 @@ import config from 'nomad-ui/config/environment';
 import * as topoScenarios from './topo';
 import * as sysbatchScenarios from './sysbatch';
 import { pickOne } from '../utils';
+import faker from 'nomad-ui/mirage/faker';
```

Comment:
`question`: They have a pinned version of `faker` before the **incident** occurred, right?

Reply:
¿ incident ?

Reply:
https://stackoverflow.com/questions/70597019/what-happened-with-faker-js

Reply:
Oh, had totally forgotten that nonsense. Yeah, we're pinned at ^4.1.0.